server/tso/allocator_manager.go

```diff
@@ -1081,8 +1084,15 @@ func (am *AllocatorManager) GetMaxLocalTSO(ctx context.Context) (*pdpb.Timestamp
 		}
 	}
 	maxTSO := &pdpb.Timestamp{}
+	if len(clusterDCLocations) == 0 {
```

Comment:
move it before line 1080?

Reply:
The loop above may change its length. So we should check after the loop.